src/rofi_rbw/clipboarder.py

```diff
@@ -49,17 +49,35 @@ def copy_to_clipboard(self, characters: str) -> None:
             encoding='utf-8'
         )
 
+        self.__last_copied_characters = characters
+
+    def fetch_clipboard_content(self) -> str:
+        return run([
+            'xsel',
+            '--output',
+            '--clipboard',
+        ],
+            capture_output=True,
+            text=True
```

Comment:
The other calls use `encoding='utf-8'`. Is there an advantage to this?

Reply:
Sorry, not that I am aware of. I have just previously used `text=True`. I have just tried it with `encoding='utf-8'` and it seems to be working as intended. Would you like me to push this change?

Reply:
Yep, that would be nice for consistency. Thanks :)